src/ctapipe/tools/calculate_pixel_stats.py

```diff
@@ -0,0 +1,185 @@
+"""
+Perform statistics calculation from pixel-wise image data
+"""
+
+import pathlib
+
+import numpy as np
+from astropy.table import vstack
+
+from ctapipe.core import Tool
+from ctapipe.core.tool import ToolConfigurationError
+from ctapipe.core.traits import (
+    Bool,
+    CInt,
+    Path,
+    Set,
+    Unicode,
+    classes_with_traits,
+)
+from ctapipe.instrument import SubarrayDescription
+from ctapipe.io import write_table
+from ctapipe.io.tableloader import TableLoader
+from ctapipe.monitoring.calculator import PixelStatisticsCalculator
+
+
+class StatisticsCalculatorTool(Tool):
+    """
+    Perform statistics calculation for pixel-wise image data
+    """
+
+    name = "StatisticsCalculatorTool"
+    description = "Perform statistics calculation for pixel-wise image data"
+
+    examples = """
+    To calculate statistics of pixel-wise image data files:
+
+    > ctapipe-calculate-pixel-statistics --input_url input.dl1.h5 --output_path /path/monitoring.h5 --overwrite
+
+    """
+
+    input_url = Path(
+        help="Input CTA HDF5 files including pixel-wise image data",
+        allow_none=True,
+        exists=True,
+        directory_ok=False,
+        file_ok=True,
+    ).tag(config=True)
```

Comment:
You should remove this option, and instead use the parameter provided by TableLoader.